babun-core/plugins/shell-here/install.sh

```diff
@@ -0,0 +1,22 @@
+set -e -f -o pipefail
+source "/usr/local/etc/babun.instance"
+source "$babun_tools/script.sh"
+
+babun_root=$( cygpath -w "/" | sed "s#\\\cygwin##g" )
+name="Babun Shell here"
+cmd="${babun_root}\cygwin\bin\mintty.exe /bin/env CHERE_INVOKING=1 /bin/zsh.exe"
+
+keys=("HKCU\Software\Classes\Directory\Background\shell\babun"
+	"HKCU\Software\Classes\Directory\shell\babun"
+	"HKCU\Software\Classes\Drive\Background\Shell\babun"
+	"HKCU\Software\Classes\Drive\shell\babun")
+
+#start with installing chere
+pact install chere || echo "Installing 'chere' failed"
+
+#install registry keys
+for key in ${keys[*]}
+do 
+	cmd /c "reg" "add" "${key}" "/ve" "/d" "${name}" "/t" "REG_SZ" "/f" || echo "Failed adding ${key}"
+	cmd /c "reg" "add" "${key}\command" "/ve" "/d" "${cmd}" "/t" "REG_EXPAND_SZ" "/f" || echo "Failed adding ${key}"
```

Comment:
Yes, this overrides the registry entries. I'm using `/f` to force changes - between installations. The latest instance is working with the `openshell here`

To differentiate instance I see to problems
* no uninstall mechanism for plugins, so when removing babun it's required to manually uninstall registry entries (otherwise some orphan entries remain)
* its not possible to differentiate the Babun installations (unless babun_root) is used. And `Open C:\Babun\ Shell here` does look very nerdish.

What I can add is a toggle: try to add registry entry and they if exists - ask for override (and use force switch than). I don't find it ideal - but something like this posts into my mind.